util/k8sutil/k8sutil.go

```diff
@@ -426,6 +426,11 @@ func (k *K8sutil) CreateClientService() error {
 				},
 			},
 		}
+		if nodePort > 0 {
+			clientSvc.Spec.ClusterIP = v1.ClusterIPNone
```

Comment:
Need to delete this line. There's a conflict when you set type to `NodePort` then set clusterIP to none.